#!/bin/sh -e
## 10_sfinfo_no_options by Daniel Kobras <kobras@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: The documented options are not implemented.

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch}"

case "$1" in
       -patch) patch $patch_opts -p1 < $0;;
       -unpatch) patch $patch_opts -p1 -R < $0;;
        *)
                echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
                exit 1;;
esac

exit 0

@DPATCH@
Index: sfinfo
===================================================================
RCS file: /home/kobras/cvsroot/debian/audiofile/docs/sfinfo,v
retrieving revision 1.1.1.1
retrieving revision 1.2
diff -u -r1.1.1.1 -r1.2
--- audiofile-0.2.6/docs/sfinfo	2001/03/13 15:42:28	1.1.1.1
+++ audiofile-0.2.6/docs/sfinfo	2001/03/14 12:10:36	1.2
@@ -1,9 +1,3 @@
-sfinfo [options] file1 file2...
+sfinfo file1 file2...
 
-options can be any combination of -reporterror and -short.
-
 sfinfo prints information regarding the file in question.
-
--short prints one-line information.
-
--reporterror prints errors if files are not valid audio files.
